Assets/Game/Addons/ModSupport/ModManager.cs

```diff
@@ -948,27 +948,36 @@ public static string MakeRelativePath(string path0, string path1)
         /// This method replicates the same behaviour for mods, doing all the hard work at build time.
         /// Results are stored to json manifest file for performant queries at runtime.
         /// </remarks>
-        public static void SeekModContributes(ModInfo modInfo)
+        public static void SeekModContributes(ModInfo modInfo, bool automaticallyRegisterQuestLists)
```

Comment:
Could have used a defaulted =false, but worst case it breaks someone's Editor tools, it doesn't break any mods

Reply:
Fixed